.pre-commit-config.yaml

```diff
@@ -36,18 +36,35 @@ repos:
       - id: rst-backticks
         # these exclusions should be removed and the files fixed
         exclude: (?x)(
-          categorical\.rst|
-          contributing\.rst|
-          contributing_docstring\.rst|
-          extending\.rst|
-          ecosystem\.rst|
-          comparison_with_sql\.rst|
-          install\.rst|
-          calculate_statistics\.rst|
-          combine_dataframes\.rst|
-          v0\.|
-          v1\.0\.|
-          v1\.1\.[012])
+            text\.rst|
```

Comment:
This is adding back exclusions, we want to remove them

Reply:
Do I have to remove all excluded file names in this file ?
![image](https://user-images.githubusercontent.com/43473352/94342816-58507f00-0031-11eb-8ca2-8d5cc01009d1.png)

Reply:
You'll want to remove any files that this PR fixes (and also not add any back that were already removed)